index.js

```diff
@@ -175,8 +180,14 @@ module.exports = {
 
     // The transpiling was done on the output of `treeForAddon` < `ember-cli@2.12.0`. We need to manually transpile
     // for >= `embe-cli@2.12.0` - @dafortin 2017.06.21
-    let BabelTranspiler = require('broccoli-babel-transpiler')
-    let output = new BabelTranspiler(iconNameTree, this._getAddonOptions().babel)
+    let output = iconNameTree
+    if (this.isEmberCliAbove12) {
+      const addonOptions = this._getAddonOptions()
+      if (addonOptions && addonOptions.babel) {
```

Comment:
Won't a single check for `addonOptions.babel` suffice since if `addonOptions` doesn't exist it will also be caught by it?